packages/calcite-components/src/components/loader/loader.scss

```diff
@@ -4,9 +4,15 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-loader-font-size: Specifies the font size of the loading percentage when type is `"determinate"`.
- * @prop --calcite-loader-size: The width and height of a non-inline loader.
- * @prop --calcite-loader-size-inline: The width and height of an inline loader
- * @prop --calcite-loader-padding : Specifies the padding of the loader.
+ * @prop --calcite-loader-size: Specifies the width and height of the component when not set to inline.
+ * @prop --calcite-loader-size-inline: Specifies the width and height of the component when set to inline.
+ * @prop --calcite-loader-spacing: Specifies the the component's padding.
+ * @prop --calcite-loader-progress-color-inline: Specifies the width and height of the component when set to inline.
+ * @prop --calcite-loader-text-spacing: Specifies the component's text margin.
+ * @prop --calcite-loader-text-weight: Specifies the component's font weight.
```

Comment:
Github comments are weird tome because it shows 4 lines and you're just saying "this token" I think it the comments refers to the bottom-most line. So if we're talking about the `text-weight` token it's also suggested to be added in the same place as the text-spacing. [Here](https://github.com/Esri/calcite-design-system/issues/6695#issuecomment-2035842204)